index.js

```diff
@@ -8,11 +8,33 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+const travelerDiv = document.querySelector('div#profile')
+travelerDiv.querySelector('img').src = `${traveler.photo}`
+travelerDiv.querySelector('h2').textContent = `${traveler.name}` 
+travelerDiv.querySelector('em').textContent = `${traveler.nickname}` 
```

Comment:
great work! There are many ways of traversing the DOM, and this works just fine.

You actually do not need to string interpolate the traveler info.
Also make sure to always include an alt attribute for your images.

```
const travelerDiv = document.querySelector('div#profile')
travelerDiv.querySelector('img').src = traveler.photo
travelerDiv.querySelector('img').alt = traveler.name
travelerDiv.querySelector('h2').textContent = traveler.name 
travelerDiv.querySelector('em').textContent = traveler.nickname 
```